db/cache/addresscache.go

```diff
@@ -426,14 +444,30 @@ func (d *AddressCacheItem) UTXOs() ([]apitypes.AddressTxnOutput, *BlockID) {
 	return d.utxos, d.blockID()
 }
 
-// Metrics is a thread-safe accessor for the *dbtypes.AddressMetrics.
-func (d *AddressCacheItem) Metrics() (*dbtypes.AddressMetrics, *BlockID) {
+// HistoryChart is a thread-safe accessor for the TxHistory.
+func (d *AddressCacheItem) HistoryChart(addrChart dbtypes.HistoryChart, chartGrouping dbtypes.TimeBasedGrouping) (*dbtypes.ChartsData, *BlockID) {
 	d.mtx.RLock()
 	defer d.mtx.RUnlock()
-	if d.metrics == nil {
+
+	if int(chartGrouping) >= dbtypes.NumIntervals {
+		log.Errorf("Invalid chart grouping: %v", chartGrouping)
 		return nil, nil
 	}
-	return d.metrics, d.blockID()
+
+	var cd *dbtypes.ChartsData
+	switch addrChart {
+	case dbtypes.TxsType:
+		cd = d.history.TypeByInterval[chartGrouping]
+	case dbtypes.AmountFlow:
+		cd = d.history.AmtFlowByInterval[chartGrouping]
+	case dbtypes.TotalUnspent:
```

Comment:
This endpoint is no longer used and misleading at the very least. See https://github.com/decred/dcrdata/pull/845#issuecomment-447893775. Might want to drop it altogether.

Reply:
Great.  I noticed it wasn't used by the address page, but hadn't checked the API thoroughly.